config.json

```diff
@@ -0,0 +1,105 @@
+{
+  "files": [
```

Comment:
Should we use yaml instead of json?  (should be a trivial code change if so, worth talking about)